### PR TITLE
fix(rxDataTable): replace custom pagination with rxPagination

### PR DESCRIPTION
**Pull request by JoseVillarreal**

### CODE REVIEW

File: src/scripts/rx-data-table.src.js [hunk 1]
```diff
@@ -268,7 +272,13 @@ app.directive('rxDataTable', function ($http, $timeout, $document, $filter, $par
...
 
                     if (!_.isEmpty(classValue)) {
-                        classes[classValue] = true;
+                        if(_.isArray(classValue)) {
+                            _.map(classValue, function (className) {
+                                classes[className] = true;
+                            });
+                        } else {
+                            classes[classValue] = true;
+                        }
```
Comment (bry4n) [inline src/scripts/rx-data-table.src.js]: This change fixes the interpolation issue for ng-class/class attributes?

File: src/templates/rx-data-table.html [hunk 2]
```diff
@@ -150,8 +150,6 @@
         </div>
     </div>
     <div ng-if="!pagerObject.showAll" class="pagination-container" ng-show="pagerObject.total > 0">
-        <div class="cell"><span ng-if="!pagerObject.showAll">{{(pagerObject.last == 0) ? '0' : pagerObject.first}} - {{pagerObject.last}} of <span class='total-data-items'>{{pagerObject.total}} {{ itemName && itemName || 'Items' }}</span></span></div>
-        <div class="cell"><rx-data-table-paginate page-tracking="pagerObject"></rx-data-table-paginate></div>
-        <div class="cell"><rx-data-table-items-per-page></rx-data-table-items-per-page></div>
+        <div class="cell"><rx-paginate page-tracking="pagerObject"></rx-paginate></div>
```
Comment (bry4n) [inline src/templates/rx-data-table.html]: 👍 see you later old pagination

**Comment by bry4n** [comment]
LGTM

File: src/scripts/rx-data-table.src.js (hunk 1 above)
Comment (JoseVillarreal) [inline src/scripts/rx-data-table.src.js]: it helps with it, but the true fix for that isn't available till angular 1.4. This does allow for 2 interpolations without ng-class being impacted.

**Pull request merged**